### ISSUE TITLE
[BUG] toast notifications do not show after waking the deck from sleep

### ISSUE THREAD

**Issue by Localhorst86** [SteamDeckHomebrew/decky-loader#604]
### Please confirm

- [X] I have searched existing issues
- [X] This issue is not a duplicate of an existing one
- [X] I have checked the [common issues section in the readme file](https://github.com/SteamDeckHomebrew/decky-loader#-common-issues)
- [X] I have attached logs to this bug report (failure to include logs will mean your issue may not be responded to).

### Bug Report Description

When waking from sleep with PIN lockscreen enabled, some toast notifications (achievements, decky loader updates etc.) do not show up. They make the toast notification sound, but the notification is not visible.

Requirements:
PIN Lockscreen

Setup:
Fresh decky loader install (wiped existing setup), both stable (2.11.1) and prerelease (2.12.0-pre1), no plugins installed

Steps to reproduce:
1. Enable a PIN lockscreen for reactivating the steam deck from sleep.
2. See if toast notifications work (switch from pre-release decky loader to stable or vice versa, you should get a notification that a new decky update is available)
3. Put steam deck to sleep using the power button.
4. Once fully put to sleep, wake the device using the power button.
5. Enter your PIN
6. See if toast notifications work (switch from pre-release decky loader to stable or vice versa, you should hear the notification sound, but not see a notification)

### Expected Behaviour

Toast Notifications should work

### SteamOS version

SteamOS 3.5.17 stable

### Selected Update Channel

Prerelease

### Backend Logs

[decky.txt](https://github.com/SteamDeckHomebrew/decky-loader/files/14653834/decky.txt)

### Frontend Logs

[cef_log.previous.txt](https://github.com/SteamDeckHomebrew/decky-loader/files/14653875/cef_log.previous.txt)

**Comment by romkslrqusz** [comment]
I am experiencing this issue too, albeit in a slightly different way.
In addition to hearing notifications be seeing no toast, there are times where I wake my Steam Deck and all of those missed notification toasts appear one after another, including playing the sound.

I have not wiped my configuration yet, so I don’t know if the variable in my case is a plugin.

**Comment by AAGaming00** [comment]
please re-test without decky installed, this might just be a Steam issue

**Comment by romkslrqusz** [comment]
> please re-test without decky installed, this might just be a Steam issue

This is a plugin issue, not a Decky issue (or a SteamOS issue)

I just installed Decky on my partner’s Deck (Stable branch), the only plugins are Animation Changer, SteamGridDB, and CSS Loader with Full Suspend.
After waking with a passcode set, that system was able to receive toast notifications for a message that I sent, I did not receive a notification for their response.

**Comment by AAGaming00** [comment]
> > please re-test without decky installed, this might just be a Steam issue
> 
> This is a plugin issue, not a Decky issue (or a SteamOS issue)
I can tell you for certain this is wrong.

**Comment by romkslrqusz** [comment]
> > > please re-test without decky installed, this might just be a Steam issue
> > 
> > 
> > This is a plugin issue, not a Decky issue (or a SteamOS issue)
> 
> I can tell you for certain this is wrong. I can reproduce this issue myself with a different set of plugins.

Are you able to reproduce the issue after wiping your config, reinstalling Decky, and having no plugins installed/loaded?

**Comment by Localhorst86** [comment]
> please re-test without decky installed, this might just be a Steam issue

a few days after submitting my bug report, I decided to disable the PIN lockscreen as a workaround. Toast notifications have been working flawlessly, ever since.

I have now tried to reproduce the issue, allthough I can clarify a bit more on when the issue happens, more on that later.

1. I have reproduced the steps mentioned in my bug report, and I can verify that these steps still realiably reproduce the bug. **Toast notifications broke**
2. I then rebooted my deck and tried the same, but I altered steps 2 and 6 from my bug report, by sending a steam chat message to my steam deck from a different account on my PC. In that situation **toast notifications remained working**.
3. I then rebooted my deck, altered steps 2 and 6 another time, by instead triggering an achievement popup. Again,  **toast notifications remained working**.
4. I then uninstalled decky loader completely, rebooted my deck and tested both stock scenarios (chat notification and achievement popup).  **toast notifications remained working**

In conclusion, the only way I was able to reliably break my toast notifications after locking and unlocking once, was by triggering a decky notification, either by switching between stable and pre-release in the settings (in case of no plugins installed) or even other plugin notifications (Flatpack updater, for example. 
Now, it might not be entirely impossible that it is a Steam issue, but if it is, I have not found a way to trigger it without decky installed.

**Comment by AAGaming00** [comment]
Interesting, that's a lot more plausible, I'll look into it

**Comment by some0nebad** [comment]
I have the same issue for a long time. Disabling the PIN code helped, thanks for workaround
